clients/client/src/main/java/org/projectnessie/client/StreamingUtil.java

```diff
@@ -63,25 +82,30 @@ private StreamingUtil() {
    * <p>The {@link Stream} returned by {@code getEntriesStream(ref, OptionalInt.empty())}, if not
    * limited, returns all commit-log entries.
    *
+   * @param api The {@link NessieApiV1} to use
    * @param ref a named reference (branch or tag name) or a commit-hash
```

Comment:
This is wrong for a while now (cannot be a commit-hash anymore).
`ref` should actually be of type `Reference` nowadays

Reply:
Do you think that we should change the API signature to accept not `Stirng ref`, but `Reference ref`?
In that case, the caller would need to do this:
`Detached.of(Detached.REF_NAME),`
It looks a bit verbose.
Also, in tests we are passing `branch.getName()` - I am not sure how to construct a `Reference` based only on the branch name. The `Branch.of()` needs two parameters: `name` and `hash`